setup.py

```diff
@@ -26,4 +26,9 @@
     scripts=SCRIPTS,
     install_requires=INSTALL_REQUIREMENTS,
     classifiers=CLASSIFIERS,
+    use_scm_version={
+        "write_to": "version.txt",
+        "tag_regex": r"^(?P<prefix>v)?(?P<version>[^\+]+)(?P<suffix>.*)?$",
```

Comment:
Is this just to avoid needing to prefix version tags with “v”?

Reply:
I think it would be useful, but I'm still puzzling through how we're going to get that version number into the docker tag. We currently don't have `setup.py build` as part of our deployment process. If we did, the version.txt file would be created. I'm thinking of adding that, but that introduces some complexities for the build server.